migration/migrator/migrations/course/20240508080451_add_peer_grading_panels_table.py

```diff
@@ -0,0 +1,44 @@
+"""Migration for a given Submitty course database."""
+
+
+def up(config, database, semester, course):
+    """
+    Run up migration.
+
+    :param config: Object holding configuration details about Submitty
+    :type config: migrator.config.Config
+    :param database: Object for interacting with given database for environment
+    :type database: migrator.db.Database
+    :param semester: Semester of the course being migrated
+    :type semester: str
+    :param course: Code of course being migrated
+    :type course: str
+    """
+    database.execute("""
+        CREATE TABLE IF NOT EXISTS peer_grading_panel (
+            g_id VARCHAR(255) PRIMARY KEY,
+            autograding BOOLEAN NOT NULL DEFAULT TRUE,
+            rubric BOOLEAN NOT NULL DEFAULT TRUE,
+            files BOOLEAN NOT NULL DEFAULT TRUE,
+            solution_notes BOOLEAN NOT NULL DEFAULT TRUE,
+            discussion BOOLEAN NOT NULL DEFAULT TRUE,
+            FOREIGN KEY (g_id) REFERENCES electronic_gradeable(g_id)
```

Comment:
After careful review, I noticed that we actually have `FOREIGN KEY` here, which is stipulating that the g_id column in the peer_grading_panel table is referencing the g_id column in the electronic_gradeable table. Hence indirectly adds constraint for primary key and foreign key.
I was initially so focused on the migration matching .sql file and had tunnel vision. And of course DB-check was not failing because it was correct migration.

Reply:
Sorry for the delay.
So, @ziesski 
Are you saying that no changes from my side are required on .sql file ?
Here's the history actually : 
There was a merge conflict in .sql file due to some course_materials changes I believe, I eventually found out and resolved it manually (Hope this is not an issue here)

Let me know if you still need anything. Thanks.